vortex-ipc/src/messages.rs

```diff
@@ -1,10 +1,12 @@
 use flatbuffers::{FlatBufferBuilder, WIPOffset};
 use itertools::Itertools;
-use vortex::flatbuffers::array as fba;
+use vortex::flatbuffers::{Array, ArrayArgs, ArrayStats, ArrayStatsArgs};
```

Comment:
Again, try to leave the `fb` prefix to make it clear when we're talking about Rust types vs FlatBuffer types